scripts/gen_profiles_layer.py

```diff
@@ -3315,31 +3315,31 @@ class JsonLoader {
 // Function symbols statically exported by this layer's library //////////////////////////////////////////////////////////////////
 // Keep synchronized with VisualStudio's VkLayer_khronos_profiles.def
 
-VK_LAYER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetInstanceProcAddr(VkInstance instance, const char *pName) {
+VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetInstanceProcAddr(VkInstance instance, const char *pName) {
```

Comment:
Vulkan layers need to export function.

See how VVL does it here:
https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/de852e626f2f6e3027448eb5c2af10af26d197e1/layers/generated/chassis.cpp#L15868

See this VVL PR for full context:
https://github.com/KhronosGroup/Vulkan-ValidationLayers/commit/be666f39c462905ef245d88ff8f36bc8a8e7cf8e